client/webserver/site/src/css/main.scss

```diff
@@ -467,3 +467,9 @@ div.border1 {
 .preline {
   white-space: pre-line;
 }
+
```

Comment:
In `market.scss` there in the `.market-bal-lbl` section, there is a `top: -11px which is causing the `Balances:` text to leave the box. I see it on both firefox and chrome. Maybe this was needed before, but now it's not.

Reply:
Was on purpose, but needs updating. I'll be doing some work ont the markets page style soon.